### PR TITLE
removed db-helper, replaced db-helper in service type, statuses, serv…

### PR DESCRIPTION
**Pull request by ErikBurdett**
…ice entries and recipients and left skeleton functions in respective models.

# Description

Summary of your changes
removed db-helper, replaced db-helper in service type, statuses, service entries and recipients and left skeleton functions in respective models. Also commented anywhere I found db-helper out. 

## What work was done?
removed db-helper, replaced db-helper in service type, statuses, service entries and recipients and left skeleton functions in respective models.

Details of your changes

## Why was this work done?
to prepare for database changes.

Why?
Fixing models and routers for database and removing db-helper

## Type of change

- New feature (non-breaking change which adds functionality)?
- Bug Fix?

## Change Status

- Completed, ready to review and merge?
- In progress?
- etc.

## Checklist

- [] Endpoint tested with Postman, or Unit test.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

**Review by chance10113 — APPROVED** [review]
Looks good, necessary skeletonization work done.

**Review by ahmedseragcodes — COMMENTED** [review]
I thought the idea was to leave the respective modeling for each section (programs, service types, service entries) and remove the DB-helper that generalizes modeling for all of these? No?

File: .env.sample [hunk 1]
```diff
@@ -1,9 +0,0 @@
-PORT=8000
-DS_API_URL=http://localhost:8002
-DS_API_TOKEN=SUPERSECRET
-DEV_DATABASE_URL=postgres://docker:docker@127.0.0.1:5400/api-dev
```
Comment (santorap) [inline .env.sample:4]: We likely want to remove the 2nd "docker" in line 4 and 5, but I think the instructions for setup reference this file as an example. Probably not a good idea to remove the file altogether.

File: api/household/householdRouter.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const express = require('express');
-const DB = require('../utils/db-helper');
+// const DB = require('../utils/db-helper');
```
Comment (santorap) [inline api/household/householdRouter.js:2]: If we _know_ we won't be using the old `db-helper` code, we should probably remove the code outright, not comment it out.

File: api/location/locationRouter.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 const express = require('express');
-const DB = require('../utils/db-helper');
+// const DB = require('../utils/db-helper');
```
Comment (santorap) [inline api/location/locationRouter.js:2]: Same as my comment in the householdRouter.js file, why comment it out when it is no longer needed and can be removed?

File: api/middleware/authRequired.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ const createError = require('http-errors');
 const OktaJwtVerifier = require('@okta/jwt-verifier');
 const { oktaVerifierConfig, okta } = require('../../config/okta');
 const Providers = require('../provider/providerModel');
-const DB = require('../utils/db-helper');
+// const DB = require('../utils/db-helper');
```
Comment (santorap) [inline api/middleware/authRequired.js:5]: Ditto to householdRouter.js and locationRouter.js.

File: api/program/programRouter.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 const express = require('express');
-const DB = require('../utils/db-helper');
+// const DB = require('../utils/db-helper');
```
Comment (santorap) [inline api/program/programRouter.js:2]: This one too.

File: api/recipient/recipientRouter.js [hunk 6]
```diff
@@ -72,7 +71,7 @@ router.post('/', (req, res, next) => {
 // PUT - Update recipient by ID
 // Only Admin and Program Managers can update recipient by ID
 router.put('/:id', requireAdmin, requireProgramManager, (req, res, next) => {
-  DB.update('recipients', req.params.id, req.body)
+  Recipients.update(req.params.id, req.body)
```
Comment (santorap) [inline api/recipient/recipientRouter.js:74]: These changes look great!

File: api/household/householdRouter.js (hunk 2 above)
Comment (santorap) [reply api/household/householdRouter.js:2]: Kind of like what you did in `serviceEntriesRouter.js`.

File: api/serviceEntries/serviceEntriesRouter.js [hunk 7]
```diff
@@ -39,7 +38,7 @@ router.post('/', (req, res, next) => {
 });
 
 router.put('/:id', (req, res, next) => {
-  ServiceEntries.update('service_entries', req.params.id, req.body)
+  ServiceEntries.update(req.params.id, req.body)
```
Comment (santorap) [inline api/serviceEntries/serviceEntriesRouter.js:41]: These changes look good!

File: api/serviceTypes/serviceTypeRouter.js [hunk 8]
```diff
@@ -71,7 +70,7 @@ router.put('/:id', canCrudServiceType, (req, res) => {
 router.delete('/:id', canCrudServiceType, (req, res, next) => {
   const { id } = req.params;
 
-  DB.remove('service_types', id)
+  ServiceTypes.remove('service_types', id)
```
Comment (santorap) [inline api/serviceTypes/serviceTypeRouter.js:73]: These changes look good!

File: api/statuses/statusesModel.js [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+const update = async (id, updates) => {
+};
+
+module.exports = {
+  knex,
+  findAll,
+  findById,
+  create,
+  update,
+};
```
Comment (santorap) [inline api/statuses/statusesModel.js:23]: 🦴☠🦴

File: api/statuses/statusesRouter.js [hunk 10]
```diff
@@ -47,7 +47,7 @@ router.put('/:id', requireAdmin, (req, res, next) => {
 router.delete('/:id', requireAdmin, (req, res, next) => {
   const { id } = req.params;
 
-  DB.remove('statuses', id)
+  Statuses.remove(id)
```
Comment (santorap) [inline api/statuses/statusesRouter.js:50]: Good updates!

**Review by santorap — CHANGES_REQUESTED** [review]
Good updates here Erik.

The .env.sample file was deleted but I think we want to keep it in there as an example during setup.

One minor question I have is regarding some of the commented out code you have in the first few files. Is there any reason we can't just delete those single lines outright?

Also, I'd prefer if someone more familiar with the needed changes here takes a closer look at `recipientModel.js`, `serviceEntriesModel.js`, and `serviceTypeModel.js`. I understand we are skeletonizing a lot of this, but some of that stuff looks like it will be substantially similar once the new tables are built.

Is it better to leave them mostly empty like you have it or should we try to keep some of it if it will work under the new structure? If it is the latter then I'll rely on your judgement for that!

**Comment by santorap** [comment]
Michael's recent pull requests supersede these updates. No need to push them any longer.

**Pull request closed without merging**